commitlint.config.ts

```diff
@@ -558,6 +560,30 @@ module.exports = {
                         `Please use full URLs instead of #XYZ refs.`
                     ];
                 },
+    
+                'proper-revert-message': ({raw}: {raw:any}) => {
+                    let offence = false;
+                    
+                    let rawStr = convertAnyToString(raw, "raw").trim();
+                    let lineBreakIndex = rawStr.indexOf('\n')
+                    
+                    if (lineBreakIndex >= 0){
+                        // Extracting bodyStr from rawStr rather than using body directly is a 
+                        // workaround for https://github.com/conventional-changelog/commitlint/issues/3412
```

Comment:
why do we need the workaround here if the tests don't involve references?

Reply:
You are right, but shouldn't I add a test that involves references? I thought it was possible in revert messages too